tools/dotnet-linker/.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
```

Comment:
From my understanding, comments are not permitted in json files or is this a special case?
Looks like Github is flagging them in red on my end

Reply:
VSCode accepts them (this is based on a template tasks.json file VSCode created), so it should be fine